src/Examine.Test/AzureDirectoryTests/ReadOnlyTest.cs

```diff
@@ -20,13 +20,14 @@
 using Examine.LuceneEngine.MergePolicies;
 using Examine.LuceneEngine.MergeShedulers;
 using Examine.LuceneEngine.DeletePolicies;
+using Examine.RemoteDirectory;
 
 namespace Examine.Test.AzureDirectory
 {
     [TestFixture]
     public class ReadOnlyTest
     {
-        private const string ContainerName = "examine-azure-directory-test";
+        private const string ContainerName = "examine-azure-syncDirectory-test";
```

Comment:
The container name needs to be lowercase as capital letters are not valid for container names. Test passes once this change is made :)

Reply:
@nzdev renamed on my local branch, will push with other changes soon :)